internal/conf/config.go

```diff
@@ -31,6 +31,9 @@ type OperatorConfig struct {
 
 	// CAIssuerName is the name of the cert-manager issuer in the operator's namespace used for the environment setup
 	CAIssuerName string `envconfig:"CA_ISSUER_NAME" required:"true"`
+
+	// BaseISOContainerImage is the container image to run the iso-package job
+	BaseISOContainerImage string `envconfig:"BASE_ISO_CONTAINER_IMAGE" required:"true"`
```

Comment:
This change breaks the operator since you're not setting BASE_ISO_CONTAINER_IMAGE anywhere. It should be set either in the `controller_manager_config.properties` or directly in the PodSpec